staff/manuel-eranueva/bash/README.md

```diff
@@ -0,0 +1,56 @@
+# BASH
+
+Command-line interface commands typical in BASH.
+
+## pwd - path to working directory
+
+```para saber dónde estás.```
```

Comment:
move comments out of the code block section. and add demo code inside block sections, instead.